src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+  background-image: url(/src/images/star.svg);
```

Comment:
The path to the image in the `background-image` property should be relative to the CSS file. If `src/images/star.svg` is not in the same directory as the compiled CSS, the path may not resolve correctly in the browser. Consider using a relative path like `../images/star.svg`.